app/javascript/components/Greeting.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { createGreetings } from '../store/greetingsSlice';
+
+const Greeting = () => {
+  const dispatch = useDispatch();
+  const greeting = useSelector((state) => state.greetings.value);
+  const status = useSelector((state) => state.greetings.status);
+
+  useEffect(() => {
+    dispatch(createGreetings());
+  }, [dispatch]);
+
+  return (
+    <div>
+      <h1>Random Greeting:</h1>
+      {status === 'loading' ? (
+        <p>Loading...</p>
+      ) : status === 'failed' ? (
+        <p className="error-message">Error loading greeting</p>
+      ) : (
+        <p className='greeting-message'>{greeting}</p>
+      )}
```

Comment:
- [ ] Nice work implementing the `Greeting` component :clap: However, the greeting message are not displaying.  Kindly check it
![Screenshot from 2023-12-05 21-02-03](https://github.com/AliRezaBmeDu/hello-rails-react/assets/91142432/a7505a3e-2419-468a-9a88-819bd744a8fa)

Reply:
done fixed